charts/chronicle/templates/statefulset.yaml

```diff
@@ -103,8 +103,11 @@ spec:
           args:
             - |
               {{ if .Values.auth.required }}
+              {{ if and (not .Values.auth.jwks.url) (not .Values.devIdProvider.enabled) }}
```

Comment:
better to invert this logic  from "A and !B" to "A or B" and switch up the if/else.  More readable.